fastlane/Fastfile

```diff
@@ -273,6 +273,10 @@ end
 
 desc "Run automatic bump in other repos"
 lane :bump_hybrid_dependencies do |options|
+  if current_version_prerelease?
+    UI.error("Hybrids won't update automatically for PHC prerelase versions")
+    next
```

Comment:
This isn't needed cause `error` throws I believe?

Reply:
See my other comment, but `error` doesn't throw. That would be `user_error!`